django_ses/management/commands/blacklist.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# encoding: utf-8
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+
+from django_ses import models
+
+
+def _add_options(target):
+    return (
+        target(
+            '-a',
+            '--add',
+            dest='add',
+            default=False,
+            help='Adds an email to your blacklist'
+        ),
+        target(
+            '-d',
+            '--delete',
+            dest='delete',
+            default=False,
+            help='Removes an email from your blacklist'
+        ),
+        target(
+            '-l',
+            '--list',
+            dest='list',
+            default=False,
+            action='store_true',
+            help='Outputs all blacklisted emails'
+        ),
+        target(
+            '-s',
+            '--search',
+            dest='search',
+            default=False,
+            help='Search for blacklisted emails'
+        ),
+    )
+
+
+class Command(BaseCommand):
+    """Add, delete or list blacklisted email addresses"""
+
+    if hasattr(BaseCommand, 'option_list'):
+        # Django < 1.10
+        option_list = BaseCommand.option_list + _add_options(make_option)
```

Comment:
We don't support Django < 3, so I think you can remove this?

Reply:
Yes! I added it simply because I copied the command that already exists and it was done the same way. I'll open another PR to remove it from there as well.